src/packages/generator-helper/src/dmmf.ts

```diff
@@ -89,9 +92,18 @@ export namespace DMMF {
   export interface Schema {
     rootQueryType?: string
     rootMutationType?: string
-    inputTypes: InputType[]
-    outputTypes: OutputType[]
-    enums: SchemaEnum[]
+    inputObjectTypes: {
+      model?: InputType[]
+      prisma: InputType[]
+    }
```

Comment:
I don't recall exactly what you said in the meeting about something here that is not actually used 🤔 
Maybe could use a comment?